pkg/volume/projected/projected.go

```diff
@@ -295,6 +299,28 @@ func (s *projectedVolumeMounter) collectData() (map[string]volumeutil.FileProjec
 			for k, v := range downwardAPIPayload {
 				payload[k] = v
 			}
+		case source.ServiceAccountToken != nil:
```

Comment:
Do we put gates here or do they go on the API side or both.

Reply:
Added feature gate.